### DISCUSSION TITLE
Whitelist my key failed！

### DISCUSSION THREAD

**Discussion by wuzhenshen2023** [ArchGryphon9362/teslabtapi#5] [Q&A]
I found the Tesla car Bluetooth device according to this document and successfully linked it. When I whitelist my key, the following data is returned to me from the car Bluetooth I received:

FromVCSECMessage {

authenticationRequest {

requestedLevel: AUTHENTICATION_ LEVEL_ DRIVE

}

}

Not as mentioned in API documentation, return

FromVCSECMessage {

commandStatus: CommandStatus {

operationStatus: OPERATIONSTATUS_ WAIT

}

}
Does anyone have the same problem? How to solve the problem. thank you!

**Comment by LexNastin** [comment]
Hey, apologies for not replying, missed the email.

Anyways, if you read further down the getting started page, you'll see that that is a request to do automated actions or tell the car which key has just connected to it, this always gets sent to you as long as you're connected and not authenticated. You can ignore the `AuthenticationRequest` messages until you have whitelisted a key. As for not receiving the message you expected, either you did receive it and just didn't realise, or you sent an invalid message, so the car just ignored your message, to help further I'd need more details.

Once again apologies for the delay.
